simphony_ui/tests/test_ui.py

```diff
@@ -13,9 +14,11 @@ def setUp(self):
         self.application = Application()
 
     def test_multi_thread(self):
-        openfoam_wrapper = mock.Mock(spec=ABCModelingEngine)
+        openfoam_wrapper = mock.MagicMock(spec=ABCModelingEngine)
```

Comment:
With a simple Mock the tests didn't pass. I had a: TypeError "Mock" object does not support indexing.